man/fi_domain.3.md

```diff
@@ -482,19 +483,28 @@ transfer operation.
   operations, or attempt to access outside of the target memory region
   will fail, resulting in a transmit error.
 
+*Unreachable EP*
+: Unreachable endpoint is a connectionless specific scenario where transmit
+  operations are issued to unreachable target endpoints. Such scenarios include
+  no-route-to-host or down target NIC. For FI_EP_DGRAM endpoints, transmit
+  operations targeting an unreachable endpoint will have operation dropped. For
+  FI_EP_RDM, target operations targeting an unreachable endpoint will result in
+  a transmit error. A provider may choose to set the completion error code to
+  FI_EHOSTUNREACH signaling to user the target endpoint is unreachable.
```

Comment:
I'd hesitate to call out a specific error code, as that depends on getting some sort of feedback from the network or target node to be accurate.